NEWS.md

```diff
@@ -1,3 +1,16 @@
+### v1.2.0 (2022-01-11)
+
+* Removed direct usage of internal tracer.
+
+* Updated README to reference `buildSubgraphSchema` instead of deprecated method `buildFederatedSchema`.
+  * `buildFederatedSchema` was [deprecated in @apollo/federation v0.28.0](https://www.apollographql.com/docs/federation/api/apollo-subgraph/#buildsubgraphschema) in favor of `buildSubgraphSchema()`.
+
+Thanks for your contribution @CacheControl 🎉
```

Comment:
Formatting is off here making the credit association awkward. 

Would either indent two spaces so it looks associated with above bullet, just add to the end of the first bullet sentence, or make a sub-bullet.